[BUG]: AttributeError: module 'numba.core.config' has no attribute 'CAPTURED_ERRORS'

### Version

25.02

### Which installation method(s) does this occur on?

Conda

### Describe the bug.

A new version of numba was pushed out on Conda 4 days ago (0.61.0) and is potentially the cause.

```
2025-02-03T18:19:56.2121782Z =========================== short test summary info ============================
2025-02-03T18:19:56.2123146Z FAILED tests/morpheus/parsers/test_ip.py::test_netmask - AttributeError: module 'numba.core.config' has no attribute 'CAPTURED_ERRORS'
2025-02-03T18:19:56.2125078Z FAILED tests/morpheus/parsers/test_ip.py::test_hostmask - AttributeError: module 'numba.core.config' has no attribute 'CAPTURED_ERRORS'
2025-02-03T18:19:56.2126979Z FAILED tests/morpheus/parsers/test_ip.py::test_mask - AttributeError: module 'numba.core.config' has no attribute 'CAPTURED_ERRORS'
```

### Minimum reproducible example

```shell

```

### Relevant log output

<details><summary>Click here to see error details</summary><pre>

https://productionresultssa13.blob.core.windows.net/actions-results/72a7c171-2b1f-4a9f-9269-d04141efd529/workflow-job-run-8a3d2706-bb10-52fe-6f95-6477f51f441c/logs/job/job-logs.txt?rsct=text%2Fplain&se=2025-02-03T20%3A18%3A49Z&sig=euGJDk99ca8sd5qyh%2BvV4qENOvJwDY5Q0xK8%2B5uBYMA%3D&ske=2025-02-04T06%3A32%3A11Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2025-02-03T18%3A32%3A11Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2025-01-05&sp=r&spr=https&sr=b&st=2025-02-03T20%3A08%3A44Z&sv=2025-01-05

</pre></details>


### Full env printout

<details><summary>Click here to see environment details</summary><pre>

 [Paste the results of print_env.sh here, it will be hidden by default]

</pre></details>


### Other/Misc.

_No response_

### Code of Conduct

- [x] I agree to follow Morpheus' Code of Conduct
- [x] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

--- a/.github/workflows/ci_pipe.yml
+++ b/.github/workflows/ci_pipe.yml
@@ -155,7 +155,7 @@ jobs:
       matrix:
         arch: ["amd64", "arm64"]
         include:
-          - runner: linux-amd64-gpu-v100-latest-1
+          - runner: linux-amd64-gpu-l4-latest-1
             arch: "amd64"
           - runner: linux-arm64-gpu-a100-latest-1
             arch: "arm64"

--- a/ci/conda/recipes/morpheus-libs/meta.yaml
+++ b/ci/conda/recipes/morpheus-libs/meta.yaml
@@ -95,6 +95,7 @@ outputs:
         - mlflow>=2.10.0,<2.18
         - mrc
         - networkx=2.8.8
+        - numba=0.60 # Pin version to avoid https://github.com/nv-morpheus/Morpheus/issues/2166
         - numpydoc =1.5.*
         - pydantic
         - pluggy =1.3.*

--- a/ci/conda/recipes/morpheus/meta.yaml
+++ b/ci/conda/recipes/morpheus/meta.yaml
@@ -101,6 +101,7 @@ outputs:
         - mlflow>=2.10.0,<2.18
         - mrc
         - networkx=2.8.8
+        - numba=0.60 # Pin version to avoid https://github.com/nv-morpheus/Morpheus/issues/2166
         - numpydoc =1.5.*
         - pydantic
         - pluggy =1.3.*

--- a/conda/environments/all_cuda-125_arch-aarch64.yaml
+++ b/conda/environments/all_cuda-125_arch-aarch64.yaml
@@ -75,6 +75,7 @@ dependencies:
 - ninja=1.11
 - nlohmann_json=3.11
 - nodejs=18.*
+- numba=0.60
 - numexpr
 - numpydoc=1.5
 - onnx=1.15

--- a/conda/environments/all_cuda-125_arch-x86_64.yaml
+++ b/conda/environments/all_cuda-125_arch-x86_64.yaml
@@ -77,6 +77,7 @@ dependencies:
 - ninja=1.11
 - nlohmann_json=3.11
 - nodejs=18.*
+- numba=0.60
 - numexpr
 - numpydoc=1.5
 - onnx=1.15

--- a/conda/environments/dev_cuda-125_arch-aarch64.yaml
+++ b/conda/environments/dev_cuda-125_arch-aarch64.yaml
@@ -64,6 +64,7 @@ dependencies:
 - ninja=1.11
 - nlohmann_json=3.11
 - nodejs=18.*
+- numba=0.60
 - numpydoc=1.5
 - pip
 - pkg-config=0.29

--- a/conda/environments/dev_cuda-125_arch-x86_64.yaml
+++ b/conda/environments/dev_cuda-125_arch-x86_64.yaml
@@ -65,6 +65,7 @@ dependencies:
 - ninja=1.11
 - nlohmann_json=3.11
 - nodejs=18.*
+- numba=0.60
 - numpydoc=1.5
 - pip
 - pkg-config=0.29

--- a/conda/environments/examples_cuda-125_arch-aarch64.yaml
+++ b/conda/environments/examples_cuda-125_arch-aarch64.yaml
@@ -34,6 +34,7 @@ dependencies:
 - mrc=25.02
 - networkx=2.8.8
 - nodejs=18.*
+- numba=0.60
 - numexpr
 - numpydoc=1.5
 - onnx=1.15

--- a/conda/environments/examples_cuda-125_arch-x86_64.yaml
+++ b/conda/environments/examples_cuda-125_arch-x86_64.yaml
@@ -35,6 +35,7 @@ dependencies:
 - networkx=2.8.8
 - newspaper3k==0.2.8
 - nodejs=18.*
+- numba=0.60
 - numexpr
 - numpydoc=1.5
 - onnx=1.15

--- a/conda/environments/runtime_cuda-125_arch-aarch64.yaml
+++ b/conda/environments/runtime_cuda-125_arch-aarch64.yaml
@@ -29,6 +29,7 @@ dependencies:
 - mlflow>=2.10.0,<2.18
 - mrc=25.02
 - networkx=2.8.8
+- numba=0.60
 - numpydoc=1.5
 - pip
 - pluggy=1.3

--- a/conda/environments/runtime_cuda-125_arch-x86_64.yaml
+++ b/conda/environments/runtime_cuda-125_arch-x86_64.yaml
@@ -29,6 +29,7 @@ dependencies:
 - mlflow>=2.10.0,<2.18
 - mrc=25.02
 - networkx=2.8.8
+- numba=0.60
 - numpydoc=1.5
 - pip
 - pluggy=1.3

--- a/dependencies.yaml
+++ b/dependencies.yaml
@@ -396,6 +396,7 @@ dependencies:
           - mlflow>=2.10.0,<2.18 # Pin version to avoid breaking change in 2.18 to thread local variable code commit id: 5541888
           - mrc=25.02
           - networkx=2.8.8
+          - numba=0.60 # Pin version to avoid https://github.com/nv-morpheus/Morpheus/issues/2166
           - numpydoc=1.5
           - pydantic
           - python-confluent-kafka>=1.9.2,<1.10.0a0

